src/webview/html/messageAsHtml.js

```diff
@@ -5,12 +5,24 @@ import { shortTime } from '../../utils/date';
 import messageTagsAsHtml from './messageTagsAsHtml';
 import messageReactionListAsHtml from './messageReactionListAsHtml';
 
-const messageDiv = (id: number, msgClass: string, flags: Object): string =>
+const flagsStateToStringList = (flags: FlagsState, id: number): string[] => {
+  const result = [];
```

Comment:
This looks like it will be better as a `reduce`:

```js
Object.keys(flags).reduce((list, key) => {
  if (flags[key][id]) {
    list.push(key);
  }
return list;
}, []);
```

Or maybe better a filter call:
```js
Object.keys(flags).filter(key => flags[key][id]);
```

Reply:
I think a filter call is the most eloquent option here. And I'll add a test or two!